django/core/checks/templates.py

```diff
@@ -1,75 +1,16 @@
-import copy
-from collections import defaultdict
+from django.template import engines
```

Comment:
Can we push this to an inner import in `check_templates()`? It will help avoid loading the whole `django.template` package when checks *aren’t* run, such as when running some commands in production.

Yes, This wasn’t the case before, but IMO the change is worth it since we’re touching this module and it’s now only one check function.

Reply:
Sure, sounds good to me.